### PR TITLE
shell: Add machine adjustments

### PR DESCRIPTION
**Pull request by petervo**
* Remove sync from normal add machine flow.
* Only clear field error when typing
* Error if adding an existing machine

### CODE REVIEW

File: pkg/shell/machine-dialogs.js [hunk 1]
```diff
@@ -370,14 +381,26 @@ define([
                 ex = new Error(_("The IP address or host name cannot contain whitespace."));
             }
 
+            if (!ex && addr) {
+                ex = existing_error(addr);
+                disabled = true;
```
Comment (stefwalter) [inline pkg/shell/machine-dialogs.js]: What is the diasbled variable used for?

**Comment by petervo** [comment]
Restored disabled use
added machine to authentication message when syncing.

**Comment by stefwalter** [comment]
A naughty file needs updating

**Comment by stefwalter** [comment]
Needs rebasing.

**Comment by stefwalter** [comment]
Looks good. Waiting for tests.

**Pull request closed without merging**